### ISSUE TITLE
Package maintenance proposal

### ISSUE THREAD

**Issue by dstepe** [philkra/elastic-apm-laravel#72]
I’ve been a casual contributor to this project and now have need of a stable release to work with Elastic APM v7+. Rather than continuing to place the burden on @philkra to own this, I propose that the package be moved to a GitHub organization where interested people can build a community to maintain it.

If this gets traction, I am happy to get the process started. If are interested, please comment on this issue and we’ll go from there. This will be my first GitHub organization and attempt to build a community maintained project. Please bear with me if I seem to go slow and take small steps. I think we can build some great on top of the work already done on this package.

**Comment by dstepe** [comment]
I've created an organization and forked this repo into it. I'll be upfront and say that I don't have time to take full responsibility for the project so I'll be looking for some interested individuals to start helping with it. 

The project can be found at:

https://github.com/nipwaayoni/elastic-apm-laravel

I selected the name nipwaayon which means "knowledge, science" in the native language of the Miami Tribe of Oklahoma. 

I'd like to invite people here to work on the project. Open an issue on the new project if you are interested.
